gcovr/tests/add_coverages/reference/coverage_bar.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
Do the tests have to include the individual files like `coverage_{foo,bar}.json` or is it sufficient to only compare the final coverage.json?

Reply:
Agreed. These files actually are not necessary to check during tests.